edx_notifications/channels/urban_airship.py

```diff
@@ -102,18 +104,21 @@ def call_ua_push_api(self, payload):
             log.error("Urban Airship push notifications API failed with error %s", ex.message)
         return resp
 
-    def bulk_dispatch_notification(self, user_ids, msg, exclude_user_ids=None, channel_context=None):
+    def bulk_dispatch_notification(self, user_ids, msg, auth_dict, exclude_user_ids=None, channel_context=None, send_to_all=False):
         """
         Perform a bulk dispatch of the notification message to
         all user_ids that will be enumerated over in user_ids.
         :param user_ids:
         :param msg:
         :param exclude_user_ids:
         :param channel_context:
+        :param send_to_all:
         :return:
         """
         if 'tag_group' in msg.payload:
             payload = self.create_tag_group_payload(msg)
+        elif send_to_all:
```

Comment:
`send_to_all ` should be a flag in `msg.payload` not a parameter because we should not change the signature of `bulk_publish_notification_to_users ` method.